pkg/controller/hyperconverged/factory.go

```diff
@@ -5,11 +5,15 @@ import (
 	networkaddons "github.com/kubevirt/cluster-network-addons-operator/pkg/apis/networkaddonsoperator/v1alpha1"
 	networkaddonsnames "github.com/kubevirt/cluster-network-addons-operator/pkg/names"
 	hcov1alpha1 "github.com/kubevirt/hyperconverged-cluster-operator/pkg/apis/hco/v1alpha1"
+
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
+
 	kubevirt "kubevirt.io/client-go/api/v1"
 	cdi "kubevirt.io/containerized-data-importer/pkg/apis/core/v1alpha1"
+	mrv1 "kubevirt.io/machine-remediation-operator/pkg/apis/machineremediation/v1alpha1"
```

Comment:
You'll need to add machine-remediation-operator to `hack/deploy.sh` to make sure it gets picked up by ci.

Reply:
@rthallisey Will add it, but why CI succeeded in the case when it does not here?

Reply:
CI is checking that the expected list of pod were launched.   Since this isn't on that list, it will pass as long as nothing else breaks.

Reply:
Thanks for the clarification, what list do I need to update?

Reply:
you'll see it in `hack/deploy.sh`. Line [72](https://github.com/kubevirt/hyperconverged-cluster-operator/blob/master/hack/deploy.sh#L72) & [80](https://github.com/kubevirt/hyperconverged-cluster-operator/blob/master/hack/deploy.sh#L80).

Reply:
done